packages/status-screen/src/StatusScreen.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, FC, useState } from 'react';
+import './global.css';
+import './StatusScreen.css';
+import { findCollection, retrieveTenantId } from './utils/setupConfiguration';
+import { CollectionKeys, SplunkCollectionItem } from './models/splunk';
+
+const StatusScreen: FC<{ theme: string }> = ({ theme }) => {
+    const [statusItems, setStatusItem] = useState<SplunkCollectionItem[]>([]);
+
+    useEffect(() => {
+        retrieveTenantId().then((id) => {
+            findCollection().then((items: SplunkCollectionItem[]) => {
+                setStatusItem(
+                    items
+                        .filter((p) => {
```

Comment:
p?

Reply:
for predicate

Reply:
Which doesn't make sense now that I realize that it's the function name, not the element name. Mind blown, I've called it like that for years